aya/src/programs/links.rs

```diff
@@ -28,6 +31,25 @@ pub trait Link: std::fmt::Debug + 'static {
     fn detach(self) -> Result<(), ProgramError>;
 }
 
+bitflags::bitflags! {
+    /// Flags passed to [`SockOps::attach()`].
+    #[derive(Clone, Copy, Debug, Default)]
+    pub struct CgroupAttachFlags: u32 {
+        /// Allows the eBPF program to be overridden by ones in a descendent cgroup.
+        const ALLOW_OVERRIDE = BPF_F_ALLOW_OVERRIDE;
+        /// Allows multiple eBPF programs to be attached.
+        const ALLOW_MULTI = BPF_F_ALLOW_MULTI;
+        /// Replaces an existing attachment.
+        const REPLACE = BPF_F_REPLACE;
+        /// Attaches the program so as to be executed first for the cgroup.
+        const BEFORE = BPF_F_BEFORE;
+        /// Attaches the program so as to be executed last for the cgroup.
+        const AFTER = BPF_F_AFTER;
+        /// Indicates the eBPF program is referred to by its ID.
+        const ID = BPF_F_ID;
```

Comment:
This wouldn't make sense to include anyway since we use FD, not ID

Reply:
Understood.  I've removed these additional flags, since `ALLOW_MULTI` and `ALLOW_OVERRIDE` were the focus of my change.